localgov_guides.install

```diff
@@ -12,4 +12,13 @@ function localgov_guides_install() {
   $services = \Drupal::moduleHandler()->moduleExists('localgov_services_navigation');
   $topics = \Drupal::moduleHandler()->moduleExists('localgov_topics');
   localgov_guides_optional_fields_settings($services, $topics);
+  // Check if simple_sitemap module is installed.
+  $simple_sitemap = \Drupal::moduleHandler()->moduleExists('simple_sitemap');
```

Comment:
Just a thought and not a blocker - is it possible that people will be using this module on their own (non-LocalGov site) with their own site sitemap config? In which case do we want to be checking whether the profile is installed?

Reply:
First thought that jumps into my brain is to do the same as I've done with workflow, which checks if there is a workflow set first, and if there isn't then it applies it's defaults. So here first `getBundleSettings` and if there are none `setBundleSettings`?

Reply:
I take your points. I think we can come back to these specific edge cases of best assumptions on default config later and tweak.